app/models/views/application_result.rb

```diff
@@ -0,0 +1,48 @@
+module Views
+  class ApplicationResult
+
+    def initialize(application)
+      @application = application
+    end
+
+    def result
+      %w[full part none].include?(outcome) ? outcome : 'error'
+    end
+
+    def amount_to_pay
+      if evidence_or_application.amount_to_pay.present?
+        "£#{evidence_or_application.amount_to_pay.round}"
+      end
+    end
+
+    def savings
+      result_translations[@application.savings_investment_valid?.to_s]
+    end
+
+    def income
+      result_translations[%w[full part].include?(result).to_s]
+    end
+
+    private
+
+    def outcome
+      case evidence_or_application
+      when EvidenceCheck
+        evidence_or_application.outcome
+      when Application
+        evidence_or_application.application_outcome
+      end
+    end
```

Comment:
My thoughts were the same. The reason I did this is that I didn't want to worry about renaming or taking the decisions out. And at the moment we have an inconsistent naming in those 2 models. 

I'd suggest though to leave it for now (as it's a bit more complex to solve). Hopefully this will be the only view model we do it in for now and then once we create the `Decision` model and abstract things to it, things will be naturally cleaner.

Is that ok?